x-pack/plugins/apm/ftr_e2e/cypress/integration/power_user/feature_flag/infrastructure.spec.ts

```diff
@@ -31,17 +27,14 @@ describe.skip('Infrastracture feature flag', () => {
     await synthtrace.clean();
   });
 
-  beforeEach(() => {
-    cy.loginAsEditorUser();
-  });
-
   describe('when infrastracture feature is enabled', () => {
-    it('shows the flag as enabled in kibana advanced settings', () => {
-      cy.visit(settingsPath);
-
-      cy.get(infraToggle)
-        .should('have.attr', 'aria-checked')
-        .and('equal', 'true');
+    beforeEach(() => {
+      cy.loginAsEditorUser().then((resp) => {
+        // enables infrastructure view feature on advanced settings
+        cy.updateAdvancedSettings({
+          'observability:enableInfrastructureView': true,
```

Comment:
I don't think we need to call/update the setting in this case, because now is enabled by default, the test was checking that the setting was enabled plus the tab showed. Maybe we can just delete the before each in this case and keep the test to check if the tab is there. Hopefully soon will delete the FF for the Infrastructure tab.

Reply:
It's either add it here or add an `after` function on the second test to update to the default value after it was set to `false`. I thought it would be more clear to have a before function to update to the state each test expects. WDYT?

Reply:
Makes sense then, as I said soon we will get rid of the FF